temporalcli/commandsmd/commands.md

```diff
@@ -2521,19 +2521,23 @@ temporal workflow trace \
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal workflow update: Synchronously run a Workflow update handler
+### temporal workflow update: Start and wait for Updates
+An Update is a synchronous call to a Workflow Execution that can change its
+state, control its flow, and return a result.
 
-Send a message to a Workflow Execution to invoke an update handler. An update
-can change the state of a Workflow Execution and return a response:
+### temporal workflow update execute: Send an Update and wait for it to complete
+Send a message to a Workflow Execution to invoke an Update handler, and wait for
+the update to complete or fail. You can also use this to wait for an existing
+update to complete, by submitting an existing update ID.
```

Comment:
This line will be changed when https://github.com/temporalio/cli/issues/647 is implemented.